app/models/roto_scraper.rb

```diff
@@ -0,0 +1,188 @@
+class RotoScraper
+
+  POSITION_PATTERN = /\| (.+?)(\z| \|)/
+
+  def self.go
+    puts "Begin scraping team urls.."
+    # TEAMS LISTING: http://www.rotoworld.com/sports/nfl/football
+    team_urls = ["http://www.rotoworld.com/teams/clubhouse/nfl/cin/cincinnati-bengals",
+                 "http://www.rotoworld.com/teams/clubhouse/nfl/cle/cleveland-browns"]
+
```

Comment:
Write the code to support the following algorithm:

1. Visit teams listing page: http://www.rotoworld.com/sports/nfl/football
2. Find all the team links (`<a>` tags) on the team listings page
3. For each team link, get the href attribute value
4. For each team link href, build the full rotoworld team URL string
5. Add each team URL to an array of `team_urls`
6. Use `team_urls` on line 11 `team_urls.each do |team_url|`